### PR TITLE
Allow all templates to be specified via labels.

### PR DESCRIPTION
**Pull request by brndnmtthws**
cc @felixb @scalp42

### CODE REVIEW

**Comment by scalp42** [comment]
So making sure I understand this correctly:

If you have an `app.json` with only 1 exposed port and you add a label `HAPROXY 0_FRONTEND_HEAD=my-template`, `marathon-lb` will look for a template named `my-template` in the `templates` dir ?

File: marathon-update-haproxy.py [hunk 1]
```diff
@@ -747,11 +788,20 @@ def writeConfig(config, config_file):
         perms = stat.S_IMODE(os.lstat(config_file).st_mode)
     os.chmod(haproxyTempConfigFile, perms)
 
-    # Move into place
-    logger.debug("moving temp file %s to %s",
-                 haproxyTempConfigFile,
-                 config_file)
-    move(haproxyTempConfigFile, config_file)
+    # Check that config is valid
+    cmd = ['haproxy', '-f', haproxyTempConfigFile, '-c']
```
Comment (scalp42) [inline marathon-update-haproxy.py]: had a PR in work for this :+1:

**Comment by brndnmtthws** [comment]
@scalp42 yes, that's correct :)

I'll update the README with more instructions!

**Comment by scalp42** [comment]
@brndnmtthws wait the doc says differently haha.

Doc says: "you pass the actual template content directly through the label"

I thought: "you pass the name of the template in the `templates` dir of `marathon-lb`"

But I understand now, you actually pass the the content of the template (and override whatever template you would have in `templates` dir).

**Comment by brndnmtthws** [comment]
Woops, guess I misunderstood.

But yes: you pass the _actual contents of the template_.

**Comment by scalp42** [comment]
Got it :+1:

**Pull request merged**